### ISSUE TITLE
Calling Sprout Forms migration from console outputs HTML

### ISSUE THREAD

**Issue by jamesmacwhite** [verbb/formie#1023]
### Describe the bug

I am looking to switch to Formie and we have a lot of existing Sprout Forms and emails/notifications setup, because of the amount of existing data, I am calling the Sprout Forms migration process via console as a web request will undoubtably timeout with all the required form, submissions and notification to migrate.

I was looking through the Formie plugin code and can see there is a console command available to call in a migration, so I ended up doing this in a content migration:

```
Craft::$app->controllerNamespace = 'formie\console\controllers';
Craft::$app->runAction('formie/migrate/migrate-sprout-forms');
```

However I noticed the output is quite hard to follow and in console HTML is being output.

![image](https://user-images.githubusercontent.com/8067792/182372511-f186795d-5252-4265-988f-2d446f470fa2.png)

I ran the command through the general `./craft ....` method rather than in a migration and it still did the same, so I don't think it's related to the fact I've done some namespace switching to be able to call the command in a content migration.

There are also stack traces being thrown, but again these are hard to follow with all the HTML being output.

I can still just about make out the progress, but is there any way this output can be cleaned up so it's easier to view the progress on console?

For context we have over a dozen Sprout Forms, with thousands of submissions across them all.

### Form settings

N/A

### Craft CMS version

3.7.50

### Plugin version

1.6.5

**Comment by jamesmacwhite** [comment]
In addition stack traces are being encoded

![image](https://user-images.githubusercontent.com/8067792/182378027-9d8c1791-6346-466f-9837-5181f8d5f8ba.png)

e.g. `&gt;` for `->`

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.6.5",
  "...": "..."
}
```

Then run `composer update`.

**Comment by jamesmacwhite** [comment]
Confirmed now fixed, thanks, a lot easier to see the output!

**Comment by jamesmacwhite** [comment]
https://github.com/verbb/formie/commit/798a7a0eccac395e46592e4fe4e265c6d5d6f6e8 Formats the output when called via console.
